src/services/ai-server/ai-server/ai.server.service.ts

```diff
@@ -160,13 +161,28 @@ export class AiServerService {
     }
   }
 
-  public async ensureSpaceBoNIsIngested(
-    persona: IAiPersonaService
-  ): Promise<void> {
+  public async ensureBoNIsIngested(persona: IAiPersonaService): Promise<void> {
     this.logger.verbose?.(
       `AI Persona service ${persona.id} found for BOK refresh`,
       LogContext.AI_SERVER
     );
+    if (persona.engine === AiPersonaEngine.GUIDANCE) {
+      [
+        'https://alkem.io/documentation',
+        'https://www.alkemio.org',
+        'https://welcome.alkem.io',
+      ].map(url => {
```

Comment:
these sites might be useful to be provided by the config

Reply:
Ultimately, those will be coming from the database (VCs configuration) and having them here minimises the footprint of this ugly hack. If you insist, I'll move them to the config.

Reply:
if it works for now, that is ok